app/main/routes.py

```diff
@@ -121,21 +117,14 @@ def download():
                 "request_type": "download",
             },
         )
-        return send_file(
-            file_content,
-            download_name=f"download-{current_datetime}.{file_format}",
-            as_attachment=True,
-            mimetype=content_type,
-        )
+
+        return redirect(url_for("main.request_received"))
 
 
 @bp.route("/request-received", methods=["GET", "POST"])
```

Comment:
I don't think we ever POST to this endpoint, do we? Can we remove that option?

Reply:
Removed the "POST" method.